.github/workflows/ci.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     strategy:
       matrix:
         dockerbase: ["centos7", "centos8"]
+      fail-fast: false
```

Comment:
Just for my own edification, what does `fail-fast` key do?

Reply:
If `fail-fast` is `true`, then if `centos7` builder fails, then Github Actions will kill all other running builders (e.g., `centos8`).  This saves CPU cycles, but it makes it so that you can only fix one builder at a time.